zio-http/src/main/scala/zhttp/http/Http.scala

```diff
@@ -686,8 +709,12 @@ object Http {
 
   private final case class FromFunctionZIO[R, E, A, B](f: A => ZIO[R, E, B]) extends Http[R, E, A, B]
```

Comment:
don't need `FromFunctionZIO` anymore.

Reply:
Should we do it in the same PR?

Reply:
Create a separate issue for this please.